doc/Lore/lore-factions.md

```diff
@@ -198,13 +198,14 @@ Acquire supplies necessary to expand the commune, help build the commune, defend
 
 ### Hub 01
 
-*"Summary"*
+*"A holdover faction of scientists connected to the ones who may have caused the Cataclysm itself."*
 
 #### Description
-
+Hub01 is one of the computer science labs set up by XEDRA. It is the housing area for Melchior, XEDRA's experimental AI system, and thanks to Melchior, the staff were able to survive the Cataclysm mostly intact. Melchior itself became briefly sentient and hyperintelligent during the portal storms, but has now been returned to being a bit smarter than chatGPT.
 
 #### Structure
-
+This section is unfinished.
+Most interaction with Hub01 will occur through the Ancillary Bar and mercenaries, a group of Hub-affiliated personnel who are not, in fact, members of the Hub.
```

Comment:
I dont think this is going to remain truthfull forever, I do intend to let the director and other scientists inside Hub to give out quests eventually. (I didnt expect this to be merged so rapidly haha)

Reply:
I figured. Just tossed a quick stub up for now.